web/packages/teleport/src/Login/useLogin.ts

```diff
@@ -31,9 +31,7 @@ export default function useLogin() {
   // see a login form which prevents login attempts.
   // Role setting requires the user to try a successful
   // attempt at logging in to determine if private key policy was enabled.
```

Comment:
Is this comment still relevant? I think the part about "cluster wide config" should be updated.